include/seastar/core/io_queue.hh

```diff
@@ -146,7 +146,7 @@ public:
     sstring mountpoint() const;
     dev_t dev_id() const noexcept;
 
-    future<> update_shares_for_class(io_priority_class pc, size_t new_shares);
+    void update_shares_for_class(io_priority_class pc, size_t new_shares);
```

Comment:
This is public, no? We can't just change the signature.

Reply:
Well, this is public method, but not the part of the (documented) API. Shares updating API sits on reactor and it's not changed by this PR

Reply:
I meant the public API, not just "public". and io_queue is definitely internal.